src/actionlist/action-list-item.scss

```diff
@@ -67,25 +81,14 @@
     &:hover,
     &:active {
       // hide dividers
-      // stylelint-disable-next-line selector-max-specificity
+      // stylelint-disable-next-line selector-max-specificity, selector-max-compound-selectors
       .ActionList-item-label::before,
       + .ActionList-item .ActionList-item-label::before {
         visibility: hidden;
       }
     }
   }
 
-  // target contents of li if sub-item (li wraps item label + nested ul)
-  // collapse styles here
-  &.ActionList-item--hasSubItem {
-    // first child
-    > .ActionList-content {
-      &:hover {
-        background-color: var(--color-action-list-item-default-hover-bg);
-      }
-    }
-  }
```

Comment:
This is now combined with the [`&:not(.ActionList-item--hasSubItem)`](https://github.com/primer/css/pull/1830/files#diff-1012f63983e65b582649d97a326609627c08f3ae717a127979884e3955a23151R37-R39) so that both have the same behavior, including that "shrink on long press" easter egg.

Although had to disable stylelint a few more times. 😬